### PR TITLE
#250: fixed markup of active orders

### PR DESCRIPTION
**Pull request by lapinlapin**
#### What's this PR do?
Fixed markup
#### Where should the reviewer start?
/views/ActiveOrders/
#### How should this be manually tested?
https://localhost:8080
#### What are the relevant tickets?
#250 
#### Screenshots (if appropriate)
image

#### URL
Provide a link to your deploy from http://staging-ui.trusty.apasia.tech:8080/branch-250-active-order-markup

### CODE REVIEW

File: src/views/ActiveOrders/ActiveOrdersTable.vue [hunk 1]
```diff
@@ -64,6 +64,6 @@ export default {
 </script>
 <style lang="scss">
   .active-orders-table {
-    height: 100%;
+    height: 92%;
```
Comment (roma219) [inline src/views/ActiveOrders/ActiveOrdersTable.vue]: why? looks like kostyl...

File: src/views/ActiveOrders/ActiveOrdersTableItem.vue [hunk 2]
```diff
@@ -204,6 +208,8 @@ export default {
     }
   }
   .active-orders-table-item {
+    height: 3.125rem;
```
Comment (roma219) [inline src/views/ActiveOrders/ActiveOrdersTableItem.vue]: should be `min-height` in case some values are broken into two lines

**Review by roma219 — CHANGES_REQUESTED** [review]
Solid, just a few things

**Comment by lapinlapin** [comment]
image

**Review by roma219 — APPROVED** [review]

**Pull request merged**